moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -210,6 +210,19 @@ void ServoCalcs::start()
   last_sent_command_ = std::move(initial_joint_trajectory);
 
   current_state_ = planning_scene_monitor_->getStateMonitor()->getCurrentState();
+
+  // Check that all links are known to the robot
+  auto check_link_is_known = [this](const std::string& frame_name) {
+    if (!current_state_->knowsFrameTransform(frame_name))
+    {
+      std::string error_string = "Unknown frame: " + frame_name;
+      throw std::runtime_error(error_string);
```

Comment:
Can be simplified: `throw std::runtime_error{"Unknown frame: " + frame_name};` Also, I take it that changing the frames on the fly might be a possibility? Still, are there any alternatives to simple terminating a process if a frame cannot be found? Is it documented anywhere that `start()` can suddenly throw an exception?

Reply:
There is a callback to change the robot command frame on the fly later. 

I added a note about the exception in the header file. However, we might consider a change so `start()` returns a bool and we can `return false` instead of throwing?